binaryen.proj

```diff
@@ -2,6 +2,8 @@
   <PropertyGroup>
     <ClangVersion Condition="'$(ClangVersion)' == ''">clang</ClangVersion>
     <ClangPlusVersion Condition="'$(ClangPlusVersion)' == ''">clang++</ClangPlusVersion>
+    <ObjCopy Condition="'$(ClangVersion)' == 'clang-9'">/usr/lib/llvm-9/bin/llvm-objcopy</ObjCopy>
```

Comment:
Why is it like this? Do we need to handle other versions specially too?

Reply:
It's a specific workaround for a build infra problem we have. `llvm-objcopy` isn't in $PATH on the Ubuntu 18.04 image we're using (and we need Ubuntu 18.04 because Mariner uses a chroot for 16.04, which doesn't support c++17, which Binaryen needs)

We can find another way to pass the value, but we need to specify a full path to llvm-objcopy in this case

Reply:
OK, seems fine then. Maybe add a comment explaining why it's there

Reply:
Couldn't we just modify PATH to include llvm-objcopy in the AzDO .yml?